containers/jetty-http/src/test/java/org/glassfish/jersey/jetty/ExceptionTest.java

```diff
@@ -44,6 +50,19 @@ public String get(@PathParam("status") int status) {
 
     }
 
+    @Test
+    public void test400StatusCodeForIllegalSymbolsInURI() throws IOException {
+        startServer(ExceptionResource.class);
+        URI testUri = getUri().build();
+        String incorrectFragment = "&params[0]=test_status";
```

Comment:
the incorrect line here should be replaces with that from reproducer. Bucause the current one does fail for 400 but it fails even before processing by Jersey's Jetty handler, but earlier inside Jetty container so it does not test the case.